sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/python/PythonStreamingWrite.scala

```diff
@@ -60,11 +57,23 @@ class PythonStreamingWrite(
   }
 
   override def commit(epochId: Long, messages: Array[WriterCommitMessage]): Unit = {
-    pythonStreamingSinkCommitRunner.commitOrAbort(messages, epochId, false)
+    new PythonStreamingSinkCommitRunner(
```

Comment:
Does this mean we can't re-use the python process, or that's handled in different way? It's not a blocker, just wanted to understand. (This wouldn't be really good for workload which has shorter batch interval. If this is former, we'd need to file a TODO JIRA ticket to make it better.)

Reply:
Found the reason.